sorc/build_compute.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+function _usage() {
+  cat << EOF
+Builds all of the global-workflow components on compute nodes.
+
+Usage: ${BASH_SOURCE[0]} [-h][-v][-A <hpc-account>][-s gfs,gfs_forecast_only,gfs_gsi,gefs,sfs,all]
```

Comment:
I would have the script signature match build_all (no `-s`; allow any number of gfs, gefs, sfs, gsi, gdas; support `-d`, etc.).

Reply:
Agreed.  I'd like to tackle this in a follow-up, though.  I kept this script as simple as possible for the time being.  I'm also not going to document this script in RTD at this time.

Reply:
Simpler than I thought.  Done.